src/test/scala-2.11/markets/participants/strategies/TestCancellationStrategy.scala

```diff
@@ -20,7 +20,7 @@ import markets.orders.Order
 import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363386120/issues/source?bid=2921361&fileBranchId=3219434#l20)